src/main/java/com/google/api/generator/engine/ast/BUILD.bazel

```diff
@@ -10,4 +10,13 @@ java_library(
     srcs = [
         ":ast_files",
     ],
+    deps = [
+        "//src/main/java/com/google/api/generator:autovalue",
+        "//src/main/java/com/google/api/generator/engine/lexicon",
+        "@com_google_auto_value_auto_value//jar",
+        "@com_google_auto_value_auto_value_annotations//jar",
```

Comment:
can I ask why we need `auto_value` dependencies here, since we have not used @auto_value in ../ast/*.java files? Thanks!

Reply:
We moved some files that use AutoValue over to the `ast` package due to the cyclic dependencies (and they make more sense here anyway).